Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -84,9 +84,10 @@ else
 fi
 echo "BUCKET_UPLOAD = $BUCKET_UPLOAD, FORCE_BUCKET_UPLOAD = $FORCE_BUCKET_UPLOAD"
 
+PACK_UPLOAD_FLAG=false
```

Comment:
Also here let's change the name to `UPLOAD_SPECICF_PACKS`
and I think we can configure this variable from the beginning of the process, meaning when triggering the flow we can pass the `UPLOAD_SPECICF_PACKS=true` and of course by default it will be false.